superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -435,7 +435,7 @@ export default class FilterableTable extends PureComponent<
         }}
         className={`grid-cell ${this.rowClassName({ index: rowIndex })}`}
       >
-        <div>{content}</div>
+        <div style={{ width: 'inherit' }}>{content}</div>
```

Comment:
It would be better to move this to a styled component. You have it available in this file already. In general, we are trying to avoid inline styles even when they are minimal like this case.